project/Build.scala

```diff
@@ -1091,6 +1092,58 @@ object Build {
       fatalWarningsSettings ++ Seq(name := "Scala.js JUnit test runtime")
   ).withScalaJSCompiler.dependsOn(testInterface)
 
+  val commonJUnitTestSettings = commonSettings ++ fatalWarningsSettings ++ Seq(
+      publishArtifact in Compile := false,
+      parallelExecution in Test := false,
+      unmanagedSourceDirectories in Test +=
+          baseDirectory.value.getParentFile / "shared/src/test/scala",
+
+      jUnitTestArgs := Seq("-v"),
+      testOptions += Tests.Argument(TestFrameworks.JUnit, jUnitTestArgs.value: _*),
+
+      checkJUnitTestFailuresOutput in Test := {
+        val _ = (testOnly in Test).toTask(" org.scalajs.junit.failure.*").failure.value
```

Comment:
Is there a way we could check that running *any* of the failure tests does indeed fail, rather than testing that when running *all* failure tests?